docs/Readme.md

```diff
@@ -46,15 +46,6 @@ to the `Cargo.toml` file:
 quicli = "0.1"
 ```
 
-To be able to use _all_ the features,
-also add these two goodies
-just below the "quicli" line:
-
-```toml file=Cargo.toml
-structopt = "0.1"
-serde = "1"
```

Comment:
Serde case is fixée?

Reply:
Serde isn't used in the example so it'd be weird to have it added here. (Before it made sense to say "let's always add structopt and serde too".)